diff --git a/liblog/logger_write.cpp b/liblog/logger_write.cpp
index 2e6b9e86..792c2c24 100644
--- a/liblog/logger_write.cpp
+++ b/liblog/logger_write.cpp
@@ -380,7 +380,7 @@ void __android_log_write_log_message(__android_log_message* log_message) {
 int __android_log_buf_write(int log_id, int prio, const char* tag, const char* msg) {
   ErrnoRestorer errno_restorer;
 
-  if (!__android_log_is_loggable(prio, tag, ANDROID_LOG_VERBOSE)) {
+  if (!__android_log_is_loggable(prio, tag, ANDROID_LOG_VERBOSE) || prio < 5) {
     return -EPERM;
   }
 
@@ -394,7 +394,7 @@ static int __android_log_buf_vprint(int log_id, int prio,
                                     const char* tag, const char* fmt, va_list ap) {
   ErrnoRestorer errno_restorer;
 
-  if (!__android_log_is_loggable(prio, tag, ANDROID_LOG_VERBOSE)) {
+  if (!__android_log_is_loggable(prio, tag, ANDROID_LOG_VERBOSE) || prio < 5) {
     return -EPERM;
   }
 
diff --git a/liblog/properties.cpp b/liblog/properties.cpp
index f5ce5efb..245b7a9d 100644
--- a/liblog/properties.cpp
+++ b/liblog/properties.cpp
@@ -245,6 +245,9 @@ static int __android_log_level(const char* tag, size_t tag_len) {
 }
 
 int __android_log_is_loggable_len(int prio, const char* tag, size_t len, int default_prio) {
+  if (prio < 5) {
+    return 0;
+  }
   int minimum_log_priority = __android_log_get_minimum_priority();
   int property_log_level = __android_log_level(tag, len);
 
@@ -303,6 +306,9 @@ int __android_log_security() {
 #else
 
 int __android_log_is_loggable(int prio, const char*, int) {
+  if (prio < 5) {
+    return 0;
+  }
   int minimum_priority = __android_log_get_minimum_priority();
   if (minimum_priority == ANDROID_LOG_DEFAULT) {
     minimum_priority = ANDROID_LOG_INFO;
