diff --git a/config/common.mk b/config/common.mk
index 65c7154..68ce933 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -52,7 +52,7 @@ endif
 
 # Lineage-specific file
 PRODUCT_COPY_FILES += \
-    vendor/custom/config/permissions/privapp-permissions-lineagehw.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/privapp-permissions-lineagehw.xml
+    vendor/custom/config/permissions/privapp-permissions-lineagehw.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/privapp-permissions-loshw.xml
 
 # Google Photos Pixel Exclusive XML
 PRODUCT_COPY_FILES += \
